tools/destination-tester/README.md

```diff
@@ -45,98 +45,113 @@ Here is an example input file named `input_1.json`:
 
 ```json
 {
-    "create_table" : {
-        "transaction": {
-            "columns": {
-                "id": "INT",
-                "amount" : "DOUBLE",
-                "desc": "STRING"
-            },
-            "primary_key": ["id"]
+  "create_table" : {
+    "transaction": {
+      "columns": {
+        "id": "INT",
+        "amount" : "DOUBLE",
+        "desc": {"type": "STRING", "string_byte_length": 256}
+      },
+      "primary_key": ["id"]
+    },
+    "campaign": {
+      "columns": {
+        "name": {"type": "STRING", "string_byte_length": 256},
+        "num": {"type": "DECIMAL", "precision": 6, "scale": 3}
+      },
+      "primary_key": []
+    }
+  },
+  "alter_table" : {
+    "transaction":
+    {
+      "add_column": {
+        "columns": {
+          "order_id": "INT",
+          "value": {
+            "type": "DECIMAL",
+            "precision": 6,
+            "scale": 3
+          }
         },
-        "campaign": {
-            "columns": {
-                "name": "STRING",
```

Comment:
Let's keep this one like this to show that byte length is optional

Reply:
done